apps/web/components/IncreasePower.tsx

```diff
@@ -163,6 +165,33 @@ export const IncreasePower = ({
     functionName: "increasePower",
     args: [requestedAmount as bigint],
   });
+  const {
+    data: decreasePowerData,
+    write: writeDecreasePower,
+    error: errorDecreasePower,
+    status: decreasePowerStatus,
+    isLoading: decreasePowerIsLoading,
+    isError: isErrordecreasePower,
+  } = useContractWrite({
+    ...registryContractCallConfig,
+    functionName: "decreasePower",
+    args: [requestedAmount as bigint],
+  });
+  useErrorDetails(errorDecreasePower, "errorDecrease");
+
+  const { updateTransactionStatus: updateDecreasePowerTransactionStatus } =
+    useTransactionNotification(decreasePowerData);
+
+  useEffect(() => {
+    updateDecreasePowerTransactionStatus(decreasePowerStatus);
+  }, [decreasePowerStatus]);
+
+  const decreasePoweErrorName = useErrorDetails(errorDecreasePower);
+  useEffect(() => {
+    if (isErrordecreasePower && decreasePoweErrorName.errorName !== undefined) {
+      toast.error(decreasePoweErrorName.errorName);
+    }
```

Comment:
if some error occours but that dont have errorName that its possible? so we dont have notification?

Reply:
looks like there is a typo in "decreasePoweErrorName" - missing "r" in Power

Reply:
@Mati0x missing that to merge

Reply:
@Mati0x just fix the typo for now and we are good to merge